projects/sbb-esta/angular-public/breadcrumb/src/styles/_breadcrumb-base.scss

```diff
@@ -75,21 +85,32 @@
     display: flex;
   }
 
+  @include businessOnly() {
+    &:first-of-type a {
+      height: toPx($breadcrumbLabelHeight);
+    }
+
+    &:last-of-type a {
+      color: $sbbColorBlack;
+    }
+  }
+
   @include mq($from: tabletPortrait) {
     display: flex;
   }
 
   padding-left: 0;
-  height: toPx($breadcrumbHeight);
```

Comment:
the height is necessary in public (because of the dropdowns), without height, dropdown panels are too small. (See the effect here: https://angular-staging.app.sbb.ch/353/public/components/breadcrumb)

Reply:
Ah jeez :/ my CSS skills are in a "nowhere but up" state. I think I fixed this which automatically seems to also fix the horizontal alignment problem, avoiding the need for a separate PR.